tests/testthat/test-case-weight-functions.R

```diff
@@ -0,0 +1,84 @@
+library(testthat)
+source(test_path("helper-case-weights.R"))
```

Comment:
Anything called "helper" is [automatically run before tests and loaded via `load_all()`](https://blog.r-hub.io/2020/11/18/testthat-utility-belt/) so we don't need to source this. We might prefer to change the file to "setup-case-weights.R" so those objects aren't in memory when we run `load_all()`